projects/core/src/occ/adapters/organization/converters/occ-user-list-normalizer.ts

```diff
@@ -10,11 +10,11 @@ import { B2B_USER_NORMALIZER } from '../../../../organization';
 
 @Injectable()
 export class OccUserListNormalizer
-  implements Converter<Occ.UserList, EntitiesModel<B2BUser>> {
+  implements Converter<Occ.B2BUserList, EntitiesModel<B2BUser>> {
```

Comment:
Changing again? Wasn't the purpose of the first renaming to align with the swagger models?

Reply:
User is an exception, because we have b2c users already. For now  I set to correct model, at the end we can rename globally